packages/docusaurus-theme-translations/index.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const path = require('path');
+const pkg = require('./package.json');
+
+module.exports = path.join(__dirname, pkg.files[0]);
```

Comment:
maybe we should add `readDefaultCodeTranslationMessages` and expose a simpler API.

Like `getPWAPluginMessages({locale})` ?

Not sure our util should be used by other plugin authors anyway

Reply:
I don't really like the idea of having specific methods like `getPWAPluginMessages`. Wouldn't it be better to move `readDefaultCodeTranslationMessages` to the translation package? Then can get rid of the `dirPath` parameter:

```js
getDefaultCodeTranslationMessages() {
  return readDefaultCodeTranslationMessages({
    locale: currentLocale,
    name: 'theme-search-algolia',
  });
},
```

Reply:
> Wouldn't it be better to move readDefaultCodeTranslationMessages to the translation package? Then can get rid of the dirPath parameter

Yes that's exactly what I had in mind.

The `getPWAPluginMessages` would just be another method so that you don't need to pass `name: "pwa-plugin"` as string => more constrained, safer API because it does not make sense to call it with `name: "plugin-that-doesnt-exist"`

But I'm fine with just moving the `readDefaultCodeTranslationMessages` msg

Reply:
This is quite an internal implementation, I won't anticipate anyone—plugin author or site creator—to know much about this package. I'd prefer `readDefaultCodeTranslationMessages({name})` over `getPWAPluginMessages` because the latter doesn't really scale. Still, both are much better than `require('theme-translations')`—it took me a while to figure out that the `require` returns a file path ㄟ(▔,▔)ㄏ

Reply:
Done, please to re-review it.